2024-02-29 02:39:16,782 - ProjectLogger - ERROR - Error whilte saving data to path: [PATH_ALREADY_EXISTS] Path file:/C:/Users/mikos/OneDrive/Desktop/PersonalDocs/Job/Technical-Case/client_data/final_data.csv already exists. Set mode as "overwrite" to overwrite the existing path.
Traceback (most recent call last):
  File "c:\Users\mikos\OneDrive\Desktop\PersonalDocs\Job\Technical-Case\source_code\app.py", line 147, in <module>
    Final_data.write.option("header", "true").csv(save_path)
  File "C:\Users\mikos\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pyspark\sql\readwriter.py", line 1864, in csv
    self._jwrite.csv(path)
  File "C:\Users\mikos\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\py4j\java_gateway.py", line 1322, in __call__
    return_value = get_return_value(
  File "C:\Users\mikos\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pyspark\errors\exceptions\captured.py", line 185, in deco
    raise converted from None
pyspark.errors.exceptions.captured.AnalysisException: [PATH_ALREADY_EXISTS] Path file:/C:/Users/mikos/OneDrive/Desktop/PersonalDocs/Job/Technical-Case/client_data/final_data.csv already exists. Set mode as "overwrite" to overwrite the existing path.
2024-02-29 02:39:48,370 - ProjectLogger - INFO - Both dataframes read by spark
2024-02-29 02:39:48,395 - ProjectLogger - INFO - Personal info and Credit card dataframes joined
2024-02-29 02:39:48,396 - ProjectLogger - INFO - Starting dropping columns function with the following columns: ['first_name', 'last_name', 'cc_n']
2024-02-29 02:39:48,404 - ProjectLogger - INFO - The following columns ['first_name', 'last_name', 'cc_n'] have been dropped
